### PR TITLE
Oak ffc ir

### PR DESCRIPTION
**Pull request by priamb**

### CODE REVIEW

**Review by Luxonis-David — APPROVED** [review]
design end looks good

File: OAK-D-SR-POE/EL6905_OAK-D-SR-POE_Power-board/Docs/Assembly Outputs/BOM/BOM-EL6905.xlsx
Comment (Erol444) [inline OAK-D-SR-POE/EL6905_OAK-D-SR-POE_Power-board/Docs/Assembly Outputs/BOM/BOM-EL6905.xlsx:1]: @priamb did you intentionally updated OAK-D-SR-POE's boards (assembly outputs)?

File: PA0960_OAK-FFC-IR-CBA/Mechanical/PA0960.step
Comment (Erol444) [inline PA0960_OAK-FFC-IR-CBA/Mechanical/PA0960.step:1]: Move step to DigitalOcean before merging

File: DM0260_OAK-FFC-CBA/3D_Models/DM0260_R2M1E1.step
Comment (Erol444) [inline DM0260_OAK-FFC-CBA/3D_Models/DM0260_R2M1E1.step:1]: Move step to DigitalOcean before merging

**Review by Erol444 — CHANGES_REQUESTED** [review]

**Pull request merged**